interface/usergroup/user_info_ajax.php

```diff
@@ -45,8 +36,8 @@
 $errMsg='';
 $success=update_password($_SESSION['authId'], $_SESSION['authId'], $curPass, $newPass, $errMsg);
 if ($success) {
-    echo xlt("Password change successful");
+    echo "<div class='alert alert-success'>" . xlt("Password change successful") . "</div>";
```

Comment:
Hi @bradymiller  @robertdown 
Where / How did you get the font-size:70%, color and text-decoration specifications?
  interface/themes/style_flat_red.css
@@ -113,7 +113,7 @@ a:hover {
     font-weight: bold;
 }
 
-.alert {
+.alert-custom {
     font-size: 70%;
     color: #FF0000;
interface/themes/style_flat_red.css
@@ -113,7 +113,7 @@ a:hover {
     font-weight: bold;
 }
 
-.alert {
+.alert-custom {
     font-size: 70%;
     color: #FF0000;
     text-decoration: none;